opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/feign/FeignTracingAutoConfigurationTest.java

```diff
@@ -0,0 +1,48 @@
+package io.opentracing.contrib.spring.cloud.feign;
+
+import io.opentracing.contrib.spring.cloud.MockTracingConfiguration;
+import org.junit.After;
+import org.junit.Test;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Eddú Meléndez
+ */
+public class FeignTracingAutoConfigurationTest {
```

Comment:
I think we can avoid writing these tests. At least if they test that `@ConditionalOnBean(Tracer.class)` works as expected.